client/app/screens/ListScreen.js

```diff
@@ -14,33 +31,72 @@ export default class ListScreen extends React.Component {
     headerTintColor: Colors.header,
   };
 
+  componentDidMount() {
+    this.props.fetchMatchesAsync('userId');
```

Comment:
Is userId supposed to be a string here or is this a placeholder?

Reply:
This is a placeholder

Reply:
In our database schema